dbt/models/staging/dashboard_pii/stg_dashboard_pii__users.sql

```diff
@@ -12,50 +24,46 @@ renamed as (
         user_type,
         case when user_type = 'student' then user_id end as student_id,
         case when user_type = 'teacher' then user_id end as teacher_id,
-        
         case when user_type = 'teacher' then email else null end as teacher_email, -- PII!
         birthday,
         datediff(year, birthday, current_date) as age_years,
+        is_active,
+        is_urg,
         races,
-        case
-            -- If races contains 'hispanic', return 'hispanic'
+        case 
             when races like '%hispanic%' then 'hispanic'
-            
-            -- If races matches any of these specific strings, return 'no_response'
-            when
-                races in ('closed_dialog', 'nonsense', 'opt_out')
-                then 'no_response'
-            
+
             -- If races contains a comma AND is_urg return 'two or more urg'
-            when races like '%,%' and is_urg = 1 then 'two_or_more_urg'
+            when races like '%,%' and is_urg = 1 then 'two or more urg'
             
             -- If races contains a comma and not caught by case above, then urg is 0 or null, return 'two or more non urg'
-            when races like '%,%' then 'two_or_more_non_urg'
+            when races like '%,%' then 'two or more'
             
+            -- If races matches any of these specific strings, return 'no_response'
+            -- when
+            --     races in ('closed_dialog', 'nonsense', 'opt_out')
+            --     then 'n/a'
             -- If races is NULL, return NULL
-            when races is null then 'not_collected'
-            
+            -- when races is null then 'n/a'
+        
             -- Default case: return the input value
-            else races -- Additional logic may be required here
+            -- else races -- Additional logic may be required here
```

Comment:
```suggestion
            else races -- Additional logic may be required here
```
The current logic is not returning race groups other than hispanic and the two+ groups. It should bring white, black, etc.